src/main/webapp/WEB-INF/jsp/display-event.jsp

```diff
@@ -57,16 +57,13 @@
           <div class="share-wrapper">
             <h3>Share</h3>
             <div class="share-container">
-              <!-- hrefs will contain the correct link when the page is generated w/JS -->
-              <a href = "https://twitter.com/share?url=http://google.com/" target="_blank">
+              <a id="twitter-link" target="_blank">
```

Comment:
I can implement the rest of this tomorrow if you haven't already.

Reply:
Wasn't it already in your PR? I had a JS method for it earlier actually but I took it off because I saw you had it in your PR haha